pkgs/cronet_http/example/android/app/build.gradle

```diff
@@ -50,6 +41,7 @@ android {
         targetSdkVersion flutter.targetSdkVersion
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
+        multiDexEnabled true
```

Comment:
This only needs to be set for cronet_http_embedded, right?

Reply:
Quite true, as it embeds more compared to using GMS's. But if we raise `minSdkVersion` to 21 we can also get rid of this.

Reply:
[Flutter supports all the way down to API 16](https://flutter-ko.dev/development/tools/sdk/release-notes/supported-platforms) but I some unofficial stats show that usage is pretty insignificant. So feel free to make the `minSdkVersion` 21 and get rid of this.

Reply:
It's `19` now actually: https://github.com/flutter/flutter/blob/e86b8258196aeb7725b0dedc9f9b81f25547c47f/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy#L46-L47